scalafmt-tests/src/test/scala/org/scalafmt/util/GitOpsTest.scala

```diff
@@ -82,6 +90,22 @@ class GitOpsTest extends fixture.FunSuite {
       ls shouldBe empty
   }
 
+  test(
+    "lsTree should return files properly when the working directory is under the git root directory") {
+    implicit ops =>
+      val f1 = touch()
+      add(f1)
+
+      val newDir = mkDir()
+      val f2 = touch(dir = Some(newDir))
+      add(f2)
+
+      {
```

Comment:
What's with the random `{}`?

Reply:
Removed.

In case you were curious though 😆 

* I was originally using an implicit `GitOpsImp`, and thought the brackets would help scope the `ls` call from the outer `GitOpsImpl` (too much Rustlang) but it didn't 
* Then I left it because I thought it made it conveyed nicely that this set of operations was taking place inside a subdirectory